x/common/types/key_rollapp_packet.go

```diff
@@ -32,19 +32,37 @@ var (
 // AckPacket with sequence 1 (originated on the hub) and OnRecvPacket with sequence 1 (originated on the rollapp).
 // Adding the packet type guarantees uniqueness as the type differentiates the source.
 func RollappPacketKey(rollappPacket *RollappPacket) []byte {
+	return RollappPacketKey1(
+		rollappPacket.Status,
+		rollappPacket.RollappId,
+		rollappPacket.ProofHeight,
+		rollappPacket.Type,
+		rollappPacket.Packet.SourceChannel,
+		rollappPacket.Packet.Sequence,
+	)
+}
+
+// RollappPacketKey1 is an alternative override for RollappPacketKey.
+func RollappPacketKey1(
```

Comment:
what does it mean? what is alternative override? I haven't seen this pattern before